crates/ide-assists/src/handlers/add_missing_impl_members.rs

```diff
@@ -2418,6 +2418,53 @@ pub struct MyStruct;
 
 impl other_file_2::Trait for MyStruct {
     $0type Iter;
+}"#,
+        );
+    }
+
+    #[test]
+    fn test_qualify_ident_pat_in_default_members() {
+        check_assist(
+            add_missing_default_members,
+            r#"
+//- /lib.rs crate:b new_source_root:library
+pub enum State {
+    Active,
+    Inactive,
+}
+
+pub trait Checker {
+    fn check(&self) -> State;
+
+    fn is_active(&self) -> bool {
+        match self.check() {
+            State::Active => true,
+            State::Inactive => false,
```

Comment:
```suggestion
            Active => true,
            Inactive => false,
```
The test is not testing this change because it's a `Path`, not an `IdentPat`. You need to `use State::*;` then use one-identifier path.